lib/scrabble.rb

```diff
@@ -10,4 +17,31 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+  def score(element)
+    #binding.pry
+
+    if element != nil
+    e = element.upcase
+      if e.length == 1
```

Comment:
even if the length of the word is `1` you could still run it through the split/map functionality as on the next branch